src/Lykke.Service.Operations/Workflow/MarketOrderWorkflow.cs

```diff
@@ -91,39 +91,47 @@ private MarketOrderFeeModel CalculateFee(CalculateMoFeeInput input)
 
         private object SendToMe(MeMoOrderInput input)
         {
-            var marketOrderModel = new MarketOrderModel
+            try
```

Comment:
Not sure why we need try/catch scope here. Anyway, the scope is too large. Why not to place it just around ME call?